Documentation/ExtensionArchitecture/BestPractises/NamingConventions.rst

```diff
@@ -220,6 +220,27 @@ Examples for Extbase domain models and table names of an extension named `cool_s
 | :php:`Vendor\CoolShop\Domain\Model\Billing\Address` | :sql:`tx_coolshop_domain_model_billing_address` |
 +-----------------------------------------------------+-------------------------------------------------+
 
+MM tables (for multiple-multiple relations between tables) follow these rules.
+
+Extbase tables SHOULD follow this pattern:
```

Comment:
Some lines above "should" is in lowercase and bold  ("Database table names should follow this pattern"), we should stick to one formatting.

Reply:
The SHOULD (in capital letters) is explained above "Hard requirements are emphasized by using the words MUST, etc. as specified in https://www.ietf.org/rfc/rfc2119.txt", but you are right, that is currently not consistently followed. I will change this to conform to the used conventions.